include/z64cutscene_commands.h

```diff
@@ -39,7 +39,7 @@
 /**
  * Marks the beginning of a cutscene script.
  */
-#define CS_BEGIN_CUTSCENE(totalEntries, frameCount) CMD_W(totalEntries), CMD_W(frameCount)
+#define CS_HEADER(totalEntries, frameCount) CMD_W(totalEntries), CMD_W(frameCount)
```

Comment:
Maybe the comment can be "Describes the length and the total number of command entries for a cutscene script"